core/src/avm2/globals/flash/text/TextField.as

```diff
@@ -154,5 +156,45 @@ package flash.text {
             stub_method("flash.text.TextField", "getCharBoundaries");
             return new Rectangle(0, 0, 1, 1);
         }
+
+        public function getFirstCharInParagraph(charIndex:int):int {
+            stub_method("flash.text.TextField", "getFirstCharInParagraph");
+            return 0;
+        }
+
+        public function getImageReference(id:String):DisplayObject {
+            stub_method("flash.text.TextField", "getImageReference");
+            return null;
+        }
+
+        public function getLineIndexAtPoint(x:Number, y:Number):int {
+            stub_method("flash.text.TextField", "getLineIndexAtPoint");
+            return 0;
+        }
+
+        public function getLineIndexOfChar(charIndex:int):int {
+            stub_method("flash.text.TextField", "getLineIndexOfChar");
+            return 0;
+        }
+
+        public function getLineOffset(lineIndex:int):int {
+            stub_method("flash.text.TextField", "getLineOffset");
+            return 0;
+        }
+
+        public function getParagraphLength(charIndex:int):int {
+            stub_method("flash.text.TextField", "getParagraphLength");
+            return 0;
+        }
+
+        public static function isFontCompatible(fontName:String, fontStyle:String):Boolean {
+            stub_method("flash.text.TextField", "isFontCompatible");
+            return true;
```

Comment:
We should probably return `false` here

Reply:
I think it's better to return true - if we return `false` and a SWF checks for this every font it uses, it might end up thinking that there aren't any available fonts.